### PR TITLE
subsume methods from AttributesKeys to AttributeKey

### PR DESCRIPTION
**Pull request by imavroukakis**
Closes https://github.com/open-telemetry/opentelemetry-java/issues/1710

### CODE REVIEW

**Review by jkwatson — APPROVED** [review]
Thanks! I had intended to do this, but it slipped my mind, so much appreciated for making it happen.

**Comment by imavroukakis** [comment]
> Thanks! I had intended to do this, but it slipped my mind, so much appreciated for making it happen.

You're very welcome!

File: api/src/main/java/io/opentelemetry/common/AttributeKey.java [hunk 1]
```diff
@@ -25,4 +26,44 @@
   /** Returns the type of attribute for this key. Useful for building switch statements. */
   @Nonnull
   AttributeType getType();
+
+  /** Create a new AttributeKey for String valued attributes. */
```
Comment (anuraaga) [inline api/src/main/java/io/opentelemetry/common/AttributeKey.java:30]: Let's change these to `Returns a` while here.

**Review by anuraaga — APPROVED** [review]
Thanks!

File: api/src/main/java/io/opentelemetry/common/AttributeKey.java (hunk 1 above)
Comment (imavroukakis) [reply api/src/main/java/io/opentelemetry/common/AttributeKey.java]: Done

**Comment by jkwatson** [comment]
I'm going to close and open this to hopefully kick the build

**Comment by imavroukakis** [comment]
> I'm going to close and open this to hopefully kick the build

If it's not too much hassle, could you do the same for https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/1307 ?

**Comment by jkwatson** [comment]
> > I'm going to close and open this to hopefully kick the build
> 
> If it's not too much hassle, could you do the same for [open-telemetry/opentelemetry-java-instrumentation#1307](https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/1307) ?

looks like that might be something different. @trask mentioned something about jfrog retention on that one.

**Comment by imavroukakis** [comment]
> > > I'm going to close and open this to hopefully kick the build
> > 
> > 
> > If it's not too much hassle, could you do the same for [open-telemetry/opentelemetry-java-instrumentation#1307](https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/1307) ?
> 
> looks like that might be something different. @trask mentioned something about jfrog retention on that one.

Ah just saw it, thanks!

**Pull request merged**